earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -25,6 +25,7 @@ sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/g
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
```

Comment:
I didn't choose the path, just followed the precedent already established in the script. We can write a follow-up ticket to change it or make it configurable. But, I like not using the default asset root for running the test. It makes not difference to me where the path is. Should I write a ticket? Do you have a suggestion for a better path?